webapi/Models/Response/FrontendConfig.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Text.Json.Serialization;
+
+namespace CopilotChat.WebApi.Models.Response;
+
+/// <summary>
+/// Configuration to be used by the frontend client to this service.
+/// </summary>
+public class FrontendConfig
+{
+    /// <summary>
+    /// Type of auth to use.
+    /// </summary>
+    [JsonPropertyName("authType")]
+    public string AuthType { get; set; } = "AzureAd";
```

Comment:
we shouldn't need a default for this since `ChatAuthenticationOptions.Type` already specifies one, but if we do keep it let's use "None" for consistency:
```suggestion
    public string AuthType { get; set; } = ChatAuthenticationOptions.AuthenticationType.None.ToString();
```